packages/synthetics-sdk-broken-links/package.json

```diff
@@ -41,6 +40,6 @@
   },
   "dependencies": {
     "@google-cloud/synthetics-sdk-api": "google-cloud-synthetics-sdk-api-0.1.0.tgz",
-    "puppeteer": "^21.0.2"
```

Comment:
What do we get from upgrading puppeteer?

Reply:
At this point nothing  but the `^` was causing us to use 21.3.6 anyway so I figured we should pin to a single version.